### PR TITLE
Label Dictionary Page

### LINKED ISSUE #55: ISIS Label Dictionary
Add to ISIS Fundamentals:

https://isis.astrogeology.usgs.gov/8.1.0/documents/LabelDictionary/LabelDictionary.html

### PR DESCRIPTION
**Pull request by amystamile-usgs**
closes #55 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

[ISIS Label Dictionary - USGS Astrogeology Software Docs.pdf](https://github.com/DOI-USGS/asc-public-docs/files/15191477/ISIS.Label.Dictionary.-.USGS.Astrogeology.Software.Docs.pdf)

### CODE REVIEW

File: docs/concepts/isis-fundamentals/label-dictionary.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    ByteOrder  = Lsb
+    Base       = 0.0
+    Multiplier = 1.0
+  EndGroup
+EndObject
+```
+
+|Keyword|Type|Description|
+|--- |--- |--- |
+|^Core|String|This optional keyword is used to point to an extra binary file containing the cube data. The cube is designated detached (labels and cube data in separate files) if this keyword exists.  If the keyword does not exist it indicates the labels and cube data are attached (in the same file).|
```
Comment (AustinSanders) [inline docs/concepts/isis-fundamentals/label-dictionary.md:72]: Is the "^" part of the keyword, or is it part of the syntax?  I don't actually know the answer here, but I was under the impression that keywords prefixed with "^" meant that the data / file exists external to the cube.

File: docs/concepts/isis-fundamentals/label-dictionary.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+|InstrumentId|String|The name of the instrument|
+|TargetName|String|The name of target/body which the instrument is viewing|
+
+## Kernels Group
+
+See [The ISIS Attached Spice Format](../isis-fundamentals/spice-format.md#the-kernels-group).
+
+## Mapping Group
+
+The Mapping Group is used to define map projections. Recommending reading includes the ISIS Map Projection Guide and Map Projections – A Working Manual, Snyder, J.P., U.S. Geological Survey, Bulletin 1532.
```
Comment (AustinSanders) [inline docs/concepts/isis-fundamentals/label-dictionary.md:222]: Can we turn this into a link to the publication? https://pubs.usgs.gov/publication/pp1395

**Review by AustinSanders — CHANGES_REQUESTED** [review]
Quick question and one small change

File: docs/concepts/isis-fundamentals/label-dictionary.md (hunk 1 above)
Comment (amystamile-usgs) [reply docs/concepts/isis-fundamentals/label-dictionary.md:72]: This is the original docs and it has `^Core` in the table. https://isis.astrogeology.usgs.gov/8.1.0/documents/LabelDictionary/LabelDictionary.html

**Review by AustinSanders — APPROVED** [review]
Quick question and one small change

**Pull request merged**